src/k2/cmd/tpcc/transactions.h

```diff
@@ -1047,6 +1051,9 @@ class DeliveryT : public TPCCTxn
                 return _txn.query(query_cid)
                 .then([this, idx] (auto&& response) {
                     CHECK_READ_STATUS(response);
+                    if (response.records.size() == 0) {
+                        return make_exception_future<>(std::runtime_error("Order not found"));
```

Comment:
might want to keep the original logic to simply ignore the order if the order is not found as the TPCC spec mentioned.

Reply:
should be fine after discussion